cubi_tk/snappy/common.py

```diff
@@ -37,6 +23,124 @@ class CouldNotFindBioMedSheet(Exception):
     """Raised when BioMedSheet could not be found in configuration file."""
 
 
+class DummyWorkflow:
+    """Dummy workflow that does nothing."""
+
+    def __init__(self):
+        self.globals = defaultdict(str)
+
+    def __getattr__(self, _):
+        return self._catchall
+
+    def _catchall(self, *_, **__):
+        pass
+
+
+def get_available_snappy_workflows():
+    """Get all available snappy workflows in snappy_pipeline.
+
+    :return: List of ModuleInfo with individual workflow modules.
+    """
+    return list(pkgutil.iter_modules(snappy_workflows.__path__))
```

Comment:
Snappy (in apps / snappy_snake) contains the `STEP_TO_MODULE` dict, that lists all fully implemented steps (and omits other workflow objects that might be under development or for internal use only), why not just import that?